### PR TITLE
upcoming: [M3-7937] - Update Placement Groups detail and summaries

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
This PR achieves two main goals:
- Consolidate summaries styles and their data (drawers, PG detail)
- Consolidate PG detail (summary + linodes list, no tabs)
- Remove "(`Affinity Type`)" appendages

## Changes  🔄
- Introduce new `DescriptionList` component for summaries (only used for PG, meant to be expanded)
  - component, test, storybook
- Implement `DescriptionList` in PG drawers and PG detail
- Remove tabs on Placement Group Details page
- Remove (`Affinity Type`)" appendages in
  - drawers
  - breadcrumb
- Update tests

## Preview 📷

| PG Detail | Create Drawer | Assign Drawer | Edit Drawer | Storybook |
| ------- | ------- | ------- | ------- | ------- |
| Screenshot 2024-04-05 at 09 08 17 | Screen Shot 2024-04-05 at 09 09 37 | Screen Shot 2024-04-05 at 09 40 32 | Screen Shot 2024-04-05 at 09 41 55 | localhost_6006__path=_docs_components-descriptionlist--documentation (1) |

## How to test 🧪

⚠️ **Note:** Copy is still WIP (main copy, errors, notices)

### Prerequisites
- Switch your environment to ALHPA and pull code locally
- Login with user `pg-user-1` (1Password)

### Verification steps
- Navigate to http://localhost:3000/placement-groups
  - Confirm create and edit drawers
- Navigate to http://localhost:3000/placement-groups/{id}
  - Confirm detail page with summary
  - Confirm assign drawer
- Start storybook: `yarn storybook`
  - Navigate to http://localhost:6006/?path=/docs/components-descriptionlist--documentation
  - Confirm `DescriptionList` behavior and styling via using the playground

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [x] 📱 Providing mobile support
- [x] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/components/EditableText/EditableText.tsx [hunk 1]
```diff
@@ -15,15 +15,15 @@ import { TextField, TextFieldProps } from '../TextField';
 const useStyles = makeStyles<void, 'editIcon' | 'icon'>()(
   (theme: Theme, _params, classes) => ({
     button: {
-      '&:first-of-type': {
+      '&[aria-label="Save"]': {
```
Comment (abailly-akamai) [inline packages/manager/src/components/EditableText/EditableText.tsx:18]: tiny fix to tighten the edit breadcrumb button which had been given an extra let margin in a regression

**Review by carrillo-erik — APPROVED** [review]

**Comment by carrillo-erik** [comment]
Looks good on first-appearance review. I like the addition of the `DescriptionList` component. The UI looks better without the tabs, the summary view was too bland on its own. Great job!

**Review by cpathipa — APPROVED** [review]
Functionality and code LGTM! 

- [x] Confirming create and edit drawers
- [x] Confirming detail page with summary
- [x] Confirming assign drawer
- [x] Confirming `DescriptionList` component.

UX - Observations:
1.Redundant configuration data in multiple spots. 
Pasted Graphic 7

2. Can we change the column name to `Status` instead of `Linode status` to maintain consistency with the Linode's table and VPC subnets table?
image

**Comment by abailly-akamai** [comment]
Thanks @cpathipa, good observations

I will take the redundant info to UX, meanwhile I did update the status column header 👍

**Pull request merged**